src/test/java/de/tum/in/www1/artemis/service/ProgrammingExerciseScheduleServiceTest.java

```diff
@@ -86,7 +86,7 @@ private void mockStudentRepoLocks() throws URISyntaxException {
     @Test
     void shouldExecuteScheduledBuildAndTestAfterDueDate() throws Exception {
         mockStudentRepoLocks();
-        long delayMS = 800;
+        long delayMS = 1000;
```

Comment:
Why was this change necessary? :D

Reply:
Like mentioned in the PR description, this was a flaky test which I tried to fix by giving it more time to exercute